src/data.js

```diff
@@ -139,3 +139,40 @@ Namespace.ensureNamespace = function(namespace, expires) {
     });
   });
 };
+
+/**Delete expired entries */
+Namespace.expireEntries = function(parent, continuationToken) {
+  var that = this;
```

Comment:
Since you're using arrow functions (`=>`) below, this line should not be necessary.

Reply:
We are using `that.query` in the next line. So will we not need that?

Reply:
Not if you use `this.query` instead :)

Reply:
oops